index.php

```diff
@@ -0,0 +1,30 @@
+<?php include('connect.php'); ?><!DOCTYPE html>
+<html>
+  <head>
+    <title>Make Your Pizza</title>
+  </head>
+  <body>
+    <div class="container">
+      <h1>Choose Your Toppings</h1>
+      <form action="review.php" method="POST">
+        <div class="group">
+          <label for="cheese">Cheese</label><?php while($row = mysqli_fetch_assoc($results['cheeses'])) { ?>
+          <input name="cheese" type="radio" value="&lt;?php echo $row['name']; ?&gt;"><span><?php echo $row['name'] ?>: $<?php echo $row['price']; ?></span><?php } ?>
+        </div>
```

Comment:
Ah, if you were converting from jade I'm not surprised it tried to change that.  Jade processing occurs before php processing so it wouldn't know that you wanted real < or >...